test/sys/test_timerfd.rs

```diff
@@ -0,0 +1,49 @@
+use nix::errno::Errno;
+use nix::sys::time::{TimeSpec, TimeValLike};
+use nix::sys::timerfd::{ClockId, Expiration, TimerFd, TimerFlags, TimerSetTimeFlags};
+use nix::unistd::read;
+use nix::Error;
+use std::os::unix::io::AsRawFd;
+use std::time::Instant;
+
+#[test]
+pub fn test_timerfd_oneshot() {
+    let mut buffer = [0u8; 64];
```

Comment:
If buffer is never read, can it be a 0-length slice?

Reply:
So, no I cannot make it a 0-length slice, the syscall requires a buffer that is no less that 8-bytes long. If you don't you get an EINVAL.

That said 64-bytes is way too big. I changed it to use the minimum required size of 8-bytes.